Library/Homebrew/api/formula.rb

```diff
@@ -15,7 +15,7 @@ def fetch(name)
           Homebrew::API.fetch "formula/#{name}.json"
         end
 
-        sig { returns(Hash) }
+        sig { returns(T.any(Array, Hash)) }
```

Comment:
This seems odd considering we are returning the value of `.to_h`

Reply:
Agreed, removed (this was a spurious failure solved on rebase, due to recent changes in Library/Homebrew/api.rb)